lesson32-home-work-43/src/components/ContactsTable.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState} from 'react';
+import Form from './Form';
+import jsonData from '../data/userData.json';
+
+const ContactsTable = () => {
+    const [contactsData, setContactsData] = useState(jsonData);
+    const [showForm, setShowForm] = useState(false);
+
+    const handleDelete = id => {
+        const updatedContacts = contactsData.filter(contact => contact.id !== id);
+        setContactsData(updatedContacts);
+    }
+
+    const tableRows = contactsData.map((info) => {

```

Comment:
Possible into constant
But, prefer inline code, without additional constants, directly in JSX

```
<table>
        <thead>
        <tr>
          <th>Name</th>
          <th>Surname</th>
          <th>Phone</th>
          <th>Action</th>
        </tr>
        </thead>
        <tbody>
        {contactsData.map((info) => (
          <tr key={info.id}>
            <td>{info.name}</td>
            <td>{info.surname}</td>
            <td>{info.phone}</td>
            <td>
              <button
                className="remove-btn"
                onClick={() => handleDelete(info.id)}
              >
                Delete
              </button>
            </td>
          </tr>
        ))}
        </tbody>
      </table>
```